test/angular-material-spec.js

```diff
@@ -203,6 +203,22 @@
       },
 
       /**
+       * Asserts that an element has keyboard focus in the DOM.
```

Comment:
Please change this to accept a string, query selector, or Element (using getElement, like below).